WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -11,4 +12,26 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
         // Then
         XCTAssertTrue(viewModel.itemsInCart.isEmpty)
     }
+
+    func test_cart_is_collapsed_when_empty_then_not_collapsed_when_has_items() {
+        // Given
+        let viewModel = PointOfSaleDashboardViewModel(items: POSItemProviderPreview().providePointOfSaleItems(),
```

Comment:
I've used the `POSItemProviderPreview` class here following the same pattern as existing tests, but should we either should use something else, or rename this to clarify it's used as a mock?